### PR TITLE
correxion de titulos e informacion

### PR DESCRIPTION
**Pull request by edburgoss**
# Correcion de titulos e informacion

## Describe the PR content and subject

Corrección de los títulos de los post, ya que los publicados no se mostraban debido a un mal uso y se modifico un texto dado que no explicaba bien la función que se detallaba en la imagen
## Proof of completion
image

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Checklisk
 
- [x]  Ensure the code follows the project's coding standards
    - [x]  Choose descriptive and unambiguous names.
    - [x]  Make meaningful distinctions.
    - [x]  Use pronounceable names.
    - [x]  Use searchable names.
    - [x]  Replace magic numbers with named constants. (Also magic text)
    - [x]  Avoid encodings. Don’t append prefixes or type information.
- [x]  Check if the new code has proper unit tests
- [x]  Verify that the code is properly commented for understandability
    - [x]  Remove poor quality, not meaningful comments
    - [x]  Use comments on non standard code and hard
- [x]  Check if the code changes are relevant to the pull request description
- [x]  Ensure the new code does not introduce any new bugs
- [x]  Check if the pull request contains minimal and logical commits
- [x]  Check if the code changes affect performance and suggest improvements if it does
- [x]  Review if the pull request resolves the issue it is linked to
    - [ ]  If not, describe why the PR y needed/split
- [x]  Do not include extensive refactoring if not requested
- [x]  Ensure all continuous integration checks are passed
- [x]  Provide proof of completion on issue or pull request

## Summary by CodeRabbit

- **Documentation**
  - Updated blog post titles to include quotation marks for improved formatting.
  - Added subtitle to "Matriz de Riesgo - Cliente" blog post.
  - Modified titles for clarity in multiple blog posts.
  - Refined language and text clarity in blog post content.
  - Improved referencing and sentence structure in blog posts.

### CODE REVIEW

File: _posts/2024-12-27-quasar-mejora-facturas.md [hunk 1]
```diff
@@ -15,18 +15,18 @@
...
 
-Adicionalmente también se incorporaron gráficos para un análisis más visual y rápido del estado de cesión de las facturas.
+Adicionalmente se incorporaron gráficos para un análisis más visual y rápido del estado de cesión de las facturas.
 <p align="center">
   <img src="https://cdn.capitalexpress.cl/img/graficos-mercado.png" alt="información general del mercado de facturas">
 </p>
 
 ## Factura por Ejecutivo
-El nuevo módulo de `Facturas por Ejecutivo` en Quasar podrás visualizar a detalle cada factura con acuse de recibo de cada ejecutivo. Para una búsqueda más ágil puedes utilizar los filtros de ejecutivo y de clientes
+El nuevo módulo de `Facturas por Ejecutivo` en Quasar, podrás visualizar a detalle cada factura con acuse de recibo de cada ejecutivo. Para una búsqueda más ágil puedes utilizar los filtros de ejecutivo y de clientes.
```
[the remark this answers is not in the document]
Comment (edburgoss) [reply _posts/2024-12-27-quasar-mejora-facturas.md:29]: no aplica

File: _posts/2024-12-27-documentos-estadoSII.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: Simplificando el Análisis: Ahora puedes revisar el estado del SII en los documentos.
+title: "Simplificando el Análisis: Ahora puedes revisar el estado del SII en los documentos."
```
[the remark this answers is not in the document]
Comment (edburgoss) [reply _posts/2024-12-27-documentos-estadoSII.md]: resuleto

File: _posts/2024-12-27-quasar-mejora-facturas.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (edburgoss) [reply _posts/2024-12-27-quasar-mejora-facturas.md]: resuelto

File: _posts/2024-12-27-quasar-mejora-facturas.md [hunk 3]
```diff
@@ -15,18 +15,18 @@ Nos complace anunciar que el `Mercado de Facturas` ya está disponible en nuestr
...
 </p>
 
-En la tabla que mencionamos anteriormente puedes hacer clic en algún ejecutivo en el que desees para ver más a detalle sus clientes con facturas sin ceder.
+En el informe previamente mencionado, tiene la opción de seleccionar el nombre de un ejecutivo para acceder a un desglose detallado de los clientes con facturas pendientes de ceder.
 <p align="center">
   <img src="https://cdn.capitalexpress.cl/img/mercado-facturas-ejecutivo.png" alt="información general del mercado de facturas">
 </p>
 
-Adicionalmente también se incorporaron gráficos para un análisis más visual y rápido del estado de cesión de las facturas.
+Adicionalmente se incorporaron gráficos para un análisis más visual y rápido del estado de cesión de las facturas.
```
[the remark this answers is not in the document]
Comment (edburgoss) [reply _posts/2024-12-27-quasar-mejora-facturas.md]: resulto

**Review by arubilar-capex — APPROVED** [review]

**Review by rhermosilla-capex — APPROVED** [review]

**Pull request merged**